portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,43 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> comments = new ArrayList<String>();
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>A work in progress...</h1>");
+    //response.setContentType("text/html;");
+    //response.getWriter().println("<h1>A work in progress...</h1>");
+
+    comments.add("This is just to practice JSON conversion");
```

Comment:
nit; since this text is fixed, you can initialize it in a constructor etc.

Reply:
I tried initializing it outside of the function but it refused to work for some reason.

Reply:
no worries, since it's Friday let's check it in for now, we can figure out later.